src/main/java/net/darkhax/tesla/Tesla.java

```diff
@@ -1,29 +1,25 @@
 package net.darkhax.tesla;
 
-import net.darkhax.tesla.api.ITeslaConsumer;
-import net.darkhax.tesla.api.ITeslaHolder;
-import net.darkhax.tesla.api.ITeslaProducer;
+import net.darkhax.tesla.api.*;
 import net.darkhax.tesla.api.implementation.BaseTeslaContainer;
-import net.darkhax.tesla.capability.TeslaCapabilities.CapabilityTeslaConsumer;
-import net.darkhax.tesla.capability.TeslaCapabilities.CapabilityTeslaHolder;
-import net.darkhax.tesla.capability.TeslaCapabilities.CapabilityTeslaProducer;
+import net.darkhax.tesla.capability.TeslaCapabilities.*;
 import net.darkhax.tesla.lib.Constants;
 import net.minecraftforge.common.capabilities.CapabilityManager;
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.fml.common.Mod.EventHandler;
 import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
 
-@Mod(modid = Constants.MOD_ID, name = Constants.MOD_NAME, version = Constants.VERSION_NUMBER, acceptedMinecraftVersions = "[1.9.4,1.10.2]")

```

Comment:
This is automatically added by ForgeGradle when you compile the mod. It will only not be added if one already exists. I had added it previously for 1.9/1.10 because they were interchangeable.